github/compliance/report.py

```diff
@@ -300,31 +397,25 @@ def _latest_processing_date(
     )
 
 
-def target_from_component_artifact(
-    component: cm.Component,
-    artifact: cm.Resource | cm.ComponentSource,
-) -> cnudie.iter.ResourceNode | cnudie.iter.SourceNode:
-    path = (component,)
-
-    if isinstance(artifact, cm.ComponentSource):
-        return cnudie.iter.SourceNode(
-            path=path,
-            source=artifact,
-        )
-    if isinstance(artifact, cm.Resource):
-        return cnudie.iter.ResourceNode(
-            path=path,
-            resource=artifact,
-        )
-
-    raise RuntimeError(f'{artifact=}')
-
-
 class PROCESSING_ACTION(enum.Enum):
     DISCARD = 'discard'
     REPORT = 'report'
 
 
+def _issue_type_close_comment(issue_type: str) -> str:
```

Comment:
this needs to depend on target-type. also, why not simply rephrase to make it generic. the semantics will always be the same regardless the type, will it not?